tiatoolbox/data/remote_samples.yaml

```diff
@@ -109,3 +109,9 @@ files:
     url: [*modelroot, "predictions/tissue_mask/small_svs_tissue_mask.npy"]
   micronet-output:
     url: [*modelroot, "predictions/nuclei_mask/micronet-output.npy"]
+  fixed_image:
+    url: [*testdata, "registration/HE_1_level8_gray.png"]
+  moving_image:
+    url: [ *testdata, "registration/HE_2_level8_gray.png" ]
```

Comment:
These could maybe be JPEG files to make download for testing faster.

Reply:
https://github.com/TissueImageAnalytics/tiatoolbox/blob/b718c77ef780c6930b720f3b8a37e130aea1f6b4/tiatoolbox/data/remote_samples.yaml#L112-L115